### PR TITLE
AlibabaCloud LogService Exporter - support new metrics interface

### PR DESCRIPTION
**Pull request by shabicheng**
**Description:** <Describe what has changed.>

* change metrics to standard pdata.Metrics interface

**Link to tracking Issue:** <Issue number if applicable>

**Testing:** <Describe what testing was performed and which tests were added.>
* unit test has passed with coverage 100%.
* Manually tested, sending to a LogService's instance.

**Documentation:** <Describe the documentation added.>

No need to update readme.

### CODE REVIEW

**Review by kongluoxing — APPROVED** [review]
LGTM

**Review by kongluoxing — APPROVED** [review]
LGTM

**Review by anuraaga — APPROVED** [review]

File: exporter/alibabacloudlogserviceexporter/tracedata_to_logservice_test.go [hunk 1]
```diff
@@ -53,14 +53,14 @@ func TestTraceDataToLogService(t *testing.T) {
 				Key:   content.GetKey(),
 				Value: content.GetValue(),
 			})
-			fmt.Printf("%s : %s\n", content.GetKey(), content.GetValue())
+			//fmt.Printf("%s : %s\n", content.GetKey(), content.GetValue())
 		}
 		gotLogPairs = append(gotLogPairs, pairs)
 
-		fmt.Println("#################")
+		//fmt.Println("#################")
```
Comment (bogdandrutu) [inline exporter/alibabacloudlogserviceexporter/tracedata_to_logservice_test.go:60]: Please remove commented debugging code

**Comment by bogdandrutu** [comment]
@shabicheng please fix the latest comments :)

**Comment by shabicheng** [comment]
> @shabicheng please fix the latest comments :)

done

**Pull request merged**